docs/enhancements/e2e-command-line.md

```diff
@@ -0,0 +1,78 @@
+# Sub Command for E2E Results Viewing
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Objectives](#objectives)
+  * [Goals](#goals)
+  * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+  * [User Stories](#user-stories)
+* [Unresolved Questions](#unresolved-questions)
+
+## Summary
+
+End to end (e2e) tests in the Kubernetes ecosystem are built on top of ginkgo
+and output junit XML. XML is not designed for human readability and thus it is
+very hard to grok. Users can use scanner but if they have results there is no
+way for a user to upload those results to scanner to get the nice output.
+
+## Objectives
+
+Make the sonobuoy e2e output be more useful to users.
+
+### Goals
+
+- Output a list of e2e tests (filterable by passed/failed).
+- Generate YAML that can rerun failed tests.
```

Comment:
plugin specific YAML

Reply:
Also, how would this feed into gen & run ?

Reply:
This will be the first user of the operators for gen and run. Presumably this will reuse the gen operator to generate some yaml and the run operator to run it. If you were thinking of a different approach I'd be happy to hear it and take your thoughts into account as I update this.